float32 rleft
float32 thetaleft
int32 conf_left
bool is_wall_left

float32 rright
float32 thetaright
int32 conf_right
bool is_wall_right

uint32 tdiv
uint32 height
int32[] accumulator
